internal/assets/contents/activestate.yaml.tpl

```diff
@@ -1,5 +1,5 @@
 project: {{.Project}}
-{{if and (.Private) (eq .CommitID "") }}
+{{if .Private }}
```

Comment:
This is to facilitate an older use-case. Ticket to get rid of it filed here:

https://activestatef.atlassian.net/browse/DX-1895

However, the change here is still troublesome as it would cause us to inject the `private` property on EVERY private project, whereas the intend is that it only happens for headless private projects. 

I think it's fine to just drop this template portion since it no longer serves a purpose as per the linked jira ticket.